AcceptanceTests/AcceptanceTests.Model/FormData/AddParticipantDatails.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace AcceptanceTests.Model.FormData
+{
+    public class ParticipantDatails : IFormData
+    {
+        public IFormData Party { get; set; }
+        public IFormData Role { get; set; }
+        public string Email { get; set; }
+        public IFormData Title { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Organisation { get; set; }
+        public string Telephone { get; set; }
+        public string DisplayName { get; set; }
+        public IFormData Address { get; set; }
+
+        public IFormData GenerateFake()
+        {
+            Party = new DropdownListFormData();
+            Role = new DropdownListFormData();
+            FirstName = Faker.Name.First();
+            LastName = Faker.Name.Last();
+            Email = $"{FirstName}.{LastName}@{Faker.Internet.DomainName()}";
```

Comment:
can you append this with "Automation_"? Same with all the fake emails and first names?

Reply:
done.